extension/README.md

```diff
@@ -0,0 +1,52 @@
+# prototype-content-consumer - Extension
+
+### Background
+
+The project will allow users to capture data on pages and store it to catalogue and organize for later use.
+
+### Getting Started
+
+Follow the steps below to setup a development environment. This project
+expects you will be using a flavor of Unix that supports a standard node.js
+environment.
+
+1. Clone repository
+
+    `git clone git@github.com:chadwpry/prototype-content-consumer.git`
+
+2. Set NODE_ENV variable to development. It can be included in a .bash_profile or .bashrc configuration.
+
+    `$ export NODE_ENV=development`
+
+3. Install node packages
+
+    `npm install`
+
+4. Create webpack bundle
+
+    `npm run build`
+
+5. Go to chrome://extensions/ in your browser and click developer mode
+
+6. Load unpacked extensions, select the extension directory in our project and click Reload
+
+7. Go to any job page, click the extension and you should see the following in chrome console:
```

Comment:
Update this from "any job" to a specific url that returns the example response below.